cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -64,6 +66,7 @@
     private static final Path BALLERINA_HOME_PATH = RepoUtils.createAndGetHomeReposPath();
     private static final Path SETTINGS_TOML_FILE_PATH = BALLERINA_HOME_PATH.resolve(
             ProjectDirConstants.SETTINGS_FILE_NAME);
+    private static Manifest manifest = new Manifest();
```

Comment:
We can make this local to both functions right ? "pushPackages" & "pushAllPackages"

Reply:
Fixed with 8534f02